ksqldb-common/src/main/java/io/confluent/ksql/util/KsqlConfig.java

```diff
@@ -165,6 +165,11 @@ public class KsqlConfig extends AbstractConfig {
   public static final String
       KSQL_SERVICE_ID_DEFAULT = "default_";
 
+  public static final String
+      KSQL_SERVICE_NAME_CONFIG = "ksql.service.name";
+  public static final String
+      KSQL_SERVICE_NAME_DEFAULT = "default_service_name";
```

Comment:
I'm not sure if we should make this an optional value. That means using an empty value as default. What do you think? This value is not used anywhere in the ksql code.

Reply:
I believe a default name makes as it will be easier to start ksql stanalone where name is not needed in the main codebase as you mentioned.
When plugging in external authentication the (empty) name check will be performed anyway when passing the configuration